spec/features/actor_stack_spec.rb

```diff
@@ -55,6 +55,18 @@ def create(env)
       end
     end
 
+    context 'when adding permissions' do
+      before do
+        work.permissions.build(name: 'user2', type: 'person', access: 'discover')
+      end
+
+      it 'persists arbitrary ACL permissions' do
+        expect { actor.create(env) }
+          .to change {  env.curation_concern.permissions.map(&:mode).map(&:first) }
+          .to include RDF::URI('http://projecthydra.org/ns/auth/acl#Discover')
```

Comment:
Would it make sense to make this test a bit more strict by checking for the name and type along with the mode?  I'm afraid of an error that would masked by having a different discover permission already on the object.

Reply:
probably a matcher is a good idea.

Reply:
@cjcolvar what do you think about the updated version with permission matcher?

Reply:
Looks good to me!